src/content_scripts/autocorrect.js

```diff
@@ -407,8 +406,8 @@ function handleResponse(message, sender) {
     fracts = message.fracts;
     autocorrections = message.autocorrections;
     longest = message.longest;
-    symbolpatterns = IS_CHROME ? new RegExp(message.symbolpatterns) : message.symbolpatterns;
-    antipatterns = IS_CHROME ? new RegExp(message.antipatterns) : message.antipatterns;
+    symbolpatterns = IS_CHROME ? new RegExp(message.symbolpatterns, "u") : message.symbolpatterns;
```

Comment:
u seems to be "match with full Unicode", do we really need that for … and why? 

![image](https://user-images.githubusercontent.com/11966684/196792951-3e0d860c-cb16-415d-a7d7-162f90857f9b.png)

(src RegEx101)

Reply:
Ah forget it it's all about unicode characters, so of course we need it sorry.